packages/beacon-node/src/api/impl/errors.ts

```diff
@@ -47,4 +47,17 @@ export class IndexedError extends ApiError {
   }
 }
 
+/**
+ * Error thrown when an array input to an API (e.g. its body, params, etc.)
+ * contains duplicate items.
+ */
+export class ApiInputDuplicateItemsError<T> extends ApiError {
```

Comment:
why do we need a custom error for this?

Reply:
I overlooked its redundancy. I first thought this will be reused for each validation, but after extracting the validation logic into a `ensureUniqueItemsOrThrow` function, it is of no use. Removed it.